sql/core/src/test/resources/sql-tests/results/ansi/decimalArithmeticOperations.sql.out

```diff
@@ -73,19 +73,17 @@ struct<(CAST(10.300000000000000000 AS DECIMAL(21,19)) * CAST(3.00000000000000000
 -- !query 8
 select (5e36 + 0.1) + 5e36
```

Comment:
This test is to check the decimal overflow behavior. let's keep testing it: `(5e36BD + 0.1) + 5e36BD`